README.rst

```diff
@@ -147,6 +156,7 @@ Complete script
 ::
 
   sudo make base apt-openfoam apt-simphony apt-lammps apt-mayavi fix-pip
+  make apt-kratos apt-numerrin 
```

Comment:
I am going to switch them from `apt-kratos` and `apt-numerrin` to `kratos` and `numerrin`.  They both copy the libs (downloaded from a website from kratos and stored in the repository from numerrin) to $(SIMPHONYENV)/lib/.

I have also altered to the LD_LIBRARY_PATH in `$(SIMPHONYENV)/bin/active` to include $(SIMPHONYENV)/lib/.

I am still running the tests but will push after things are working locally.